common/convert/convert.go

```diff
@@ -113,3 +114,15 @@ func BoolPtr(condition bool) *bool {
 	b := condition
 	return &b
 }
+
+// StripSpaceBuilder removes spaces from string using string builder
+func StripSpaceBuilder(str string) string {
+	var b strings.Builder
+	b.Grow(len(str))
+	for _, ch := range str {
```

Comment:
because of the use of IsSpace in the unicode package its a loss of performance here to index since i have to convert it

But StripSpaceBuilder is from an earlier iteration before i was passing in full ohlcv data and was just passing in as a string 

will remove function as its no longer used

Reply:
okay!